js/screen/app/partials/_main_view.html

```diff
@@ -81,6 +83,31 @@ <h2 class="page-header">Controller</h2>
     </div>
   </div>
 
+  <div class="row">
+    <div class="col-md-12">
+      <h2 class="page-header">Saving player state</h2>
+      <ul>
+        <li class="checkbox" ng-repeat="q in questions.localisation">
+          <label>
+            <input type="checkbox" ng-checked="q.checked" ng-model="questions.localisation[$index].checked" ng-change="updateProgress(q)"> <span ng-bind-html="q.text"></span>
+          </label>
+        </li>
+      </ul>
+    </div>
+  </div>
+
+  <div class="row">
+    <div class="col-md-12">
+      <h2 class="page-header">Localisation</h2>
```

Comment:
This is the wrong heading, isn't it?

```suggestion
      <h2 class="page-header">Player state persistence</h2>
```

Reply:
✅